.github/workflows/build_test.yml

```diff
@@ -7,6 +7,30 @@ on:
     branches: [main]
 
 jobs:
+  check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check License Header
+        uses: apache/skywalking-eyes/header@main
```

Comment:
Cool find :)

Should we pin to a revision?  The example on their main page says:

```
  uses: apache/skywalking-eyes/header@main      # always prefer to use a revision instead of `main`.
```

Reply:
Fixed